### DISCUSSION TITLE
Wildcard

### DISCUSSION THREAD

**Discussion by lbtboy** [AdguardTeam/AdGuardHome#3230] [General]
Здравствуйте. 
Кто устанавливал SSL сертификат Wildcard от Let’s Encrypt. Подскажите работает ли у вас вот это https://github.com/AdguardTeam/AdGuardHome/wiki/Clients?

4.Client ID (added in v0.105). This method is supposed to be used for encrypted DNS protocols (DoH, DoT or DoQ). Once this identifier is used, you can use a special domain name while configuring your client. Here's an example:

    AdGuard Home has the domain name example.org.

    In AdGuard Home you add a client with the client ID my-client.

    On the client device you can now configure:
        DNS-over-HTTPS: https://example.org/dns-query/my-client
        DNS-over-TLS: tls://my-client.example.org (requires a Wildcard certificate)
        DNS-over-QUIC: quic://my-client.example.org (requires a Wildcard certificate)

Note that the certificate for TLS and QUIC client IDs must be valid both for *.example.org and [example.org.]
Сертификаты получены и валидны
![Screenshot 2021-06-05 at 12-12-15 SSL Server Test lbtboy ru (Powered by Qualys SSL Labs)](https://user-images.githubusercontent.com/57795912/120884709-264ede00-c5fe-11eb-8ee0-10bf54b9d9f6.png)
![Screenshot 2021-06-05 at 12-13-07 AdGuard Home](https://user-images.githubusercontent.com/57795912/120884710-27800b00-c5fe-11eb-8ff7-31f955eac2c5.png)
![Screenshot 2021-06-05 at 12-51-09 SSL TLS lbtboy ru Account Cloudflare - Web Performance Security](https://user-images.githubusercontent.com/57795912/120884711-2818a180-c5fe-11eb-95c1-ee9302e2fadd.png)
Идентификаторы клиентов добавлены, но когда прописываешь в клиенте например quic://my-phone.****oy.ru или 
tls://my-phone.****oy.ru интернет на устройстве пропадает, стоит прописать только корневой домен 
quic://****oy.ru или tls://****oy.ru все прекрасно работает. С DoH такой проблемы нет. В чем проблема помогите разобратся.

**Comment by ameshkov** [comment]
Я вам рекомендую отлаживать доступность с помощью утилиты dnslookup: https://github.com/ameshkov/dnslookup

Она хотя бы ошибку покажет корректную если у вас что-то не так настроено.

**Reply by lbtboy** [reply]
Все настроенно правильно. Добавил A запись с именем * и все заработало.
![A_запись](https://user-images.githubusercontent.com/57795912/121211850-439dd980-c896-11eb-86c6-e3dc017dae78.png)
Тема закрыта
